Explorer/Assets/DCL/MapRenderer/ConsumerUtils/MapRenderImage.cs

```diff
@@ -175,6 +175,14 @@ public void Deactivate()
             hudCamera = null;
             interactivityController = null;
             texture = null;
+            previousClickedMarker?.DeselectImmediately();
+            previousClickedMarker?.ResetScale(IPinMarker.ScaleType.NAVMAP);

```

Comment:
Well, we could include it inside the DeselectImmediately tbh, I didnt think about it xD
Still this method is almost exclusively used by this class because of how the map is built, but you are right :)